bucket/gimp.json

```
@@ -41,6 +41,7 @@
         "bin\\gimp-console-2.10.exe",
         [
             "bin\\gimp-console-2.10.exe",
+            "gimp-console",
             "gimp"
         ],
```

This was not requested

```suggestion
            "gimp"
        ],
        [
            "bin\\gimp-console-2.10.exe",
            "gimp-console"
        ],
```